cylc/uiserver/resolvers.py

```diff
@@ -251,11 +255,9 @@ async def clean(
                 executor, _clean, workflow_ids, opts
             )
         except Exception as exc:
-            if isinstance(exc, ServiceFileError):  # Expected error
-                # The "workflow still running" msg is very long
-                msg = str(exc).split('\n')[0]
```

Comment:
The error caught here is not `ServiceFileError`, but a `CylcError`: see https://github.com/cylc/cylc-flow/blob/98181b6a80d89f5fbc57cbee964609bb320d3bde/cylc/flow/scripts/clean.py#L231-L240

We should rethink that error message, see https://github.com/cylc/cylc-flow/issues/5324